src/style.css

```diff
@@ -1,3 +1,70 @@
+* {
+  margin: 0;
+
+  --bluecolor: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the 'Roboto' font is correctly loaded in your HTML file. This is crucial for the font-family to be applied as expected.